modules/helfi_media_remote_video/helfi_media_remote_video.module

```diff
@@ -77,7 +77,14 @@ function _helfi_media_remote_video_remote_video_provider_validation(array $form,
     $user_input = $form_state->getUserInput();
     $allowed_providers = $config_data['source_configuration']['providers'];
     $url_resolver = Drupal::service('media.oembed.url_resolver');
-    $provider = $url_resolver->getProviderByUrl($user_input['url']);
+
+    try {
+      $provider = $url_resolver->getProviderByUrl($user_input['url']);
+    }
+    catch (\Exception $e) {
```

Comment:
I think we should catch `\Drupal\media\OEmbed\ResourceException` here instead

Reply:
Switched to ResourceException.